### PR TITLE
FINERACT-1452: Disallow changing of password of specific users

### PR DESCRIPTION
**Pull request by ptuomola**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by ptuomola** [comment]
@vorburger How about something as simple as this to stop the people changing the password? When setting up the instance you could just run "update fineract_default.m_appuser set cannot_change_password = 1 where id = 1;" to enable this for the admin user.

**Comment by vorburger** [comment]
This looks very interesting, for https://github.com/vorburger/www.fineract.dev/issues/5.

I'm going to review this PR, merge it when it LGTM, and use it on Fineract.dev...

File: fineract-provider/src/main/resources/sql/migrations/core_db/V379__user_cannot_change_password.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+--
```
Comment (vorburger) [inline fineract-provider/src/main/resources/sql/migrations/core_db/V379__user_cannot_change_password.sql:1]: I've verified that the last one in https://github.com/apache/fineract/tree/develop/fineract-provider/src/main/resources/sql/migrations/core_db is 378, so this at 379 LGTM.

**Review by vorburger — APPROVED** [review]
This looks about right, to me. (I've not actually really tested it though.)

But please hold off merging this until I fix https://github.com/vorburger/www.fineract.dev/issues/8

**Pull request merged**